Packs/AgariPhishingDefense/Integrations/AgariPhishingDefense/AgariPhishingDefense.py

```diff
@@ -719,9 +720,10 @@ def fetch_incidents(client: Client, last_run: Dict[str, Any], args: Dict[str, An
         incidents.append(incident)
 
     # Update last run and add incident if the incident is newer than last fetch
-    latest_created_time = dateparser.parse(total_records[-1]['created_at'])
-    latest_created_time = latest_created_time.strftime(DATE_FORMAT)
-    next_run = {'last_fetch': latest_created_time, 'id': total_records[-1]['id']}
+    latest_created_time_date = dateparser.parse(total_records[-1]['created_at'])
+    assert latest_created_time_date is not None
```

Comment:
Add an error message so it is clearer why it failed
```suggestion
    assert latest_created_time_date is not None, f'failed parsing created_at date: {total_records[-1]['created_at']}'
```